apps/docs/src/app/core/component-docs/dialog/examples/stacked-dialogs/first-dialog-example.component.ts

```diff
@@ -7,7 +7,7 @@ import { SecondDialogExampleComponent } from './second-dialog-example.component'
         <fd-dialog>
             <fd-dialog-header>
                 <h1 id="fd-dialog-header-8" fd-title>First Dialog</h1>
-                <button fd-dialog-close-button (click)="dialogRef.dismiss('x')"></button>
+                <button fd-dialog-close-button (click)="dialogRef.dismiss('x')" i18n-title="@@dialogPlayGround.closebutton.title"></button>
```

Comment:
`i18n-title` not needed here as explained above. `title` is needed.

Reply:
fixed